### ISSUE TITLE
empty try structure

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1144]
```
TRY.
  CATCH cx_errror INTO something.
ENDTRY.
```

should be reported as error by https://rules.abaplint.org/empty_structure/
